examples/cpp/core/multiprecision/CMakeLists.txt

```diff
@@ -8,3 +8,7 @@ add_bond_codegen (multiprecision.bond
 add_bond_test (multiprecision
     ${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_CFG_INTDIR}/multiprecision_types.cpp
     multiprecision.cpp)
+
+# Deep in the guts of Boost we get a warning about not using a "secure"
+# routine. Suppress them for this example.
+cxx_target_compile_options(MSVC multiprecision PRIVATE -D_SCL_SECURE_NO_WARNINGS)
```

Comment:
Yeah, the Boost stuff emitting a warning is being pulled in by the example in question, not by Bond.